config/initializers/braintree.rb

```diff
@@ -1,8 +1,3 @@
 if SolidusSupport.backend_available?
   Spree::Admin::PaymentsController.helper :braintree_admin
 end
-
-if SolidusSupport.frontend_available?
-  Spree::CheckoutController.helper :braintree_checkout
-  Spree::OrdersController.helper :braintree_checkout
-end
```

Comment:
Can you deal with this in a separate PR? 
Maybe check if the backend helper above can also be removed. 🙏